src/components/AccountDrawer/MiniPortfolio/Activity/ActivityRow.tsx

```diff
@@ -43,8 +43,9 @@ function StatusIndicator({ activity: { status, timestamp } }: { activity: Activi
 }
 
 export function ActivityRow({ activity }: { activity: Activity }) {
-  const { chainId, title, descriptor, logos, otherAccount, currencies, hash, prefixIconSrc, offchainOrderStatus } =
-    activity
+  const { chainId, title, descriptor, logos, otherAccount, currencies, hash, prefixIconSrc } = activity
+  const { offchainOrderStatus, isSpam } = activity
```

Comment:
I know this pattern is wonky but I will fight for it over the alternative 🤺

Reply:
why this pattern? 🤺

Reply:
yeah, what's the benefit of destructuring these separately? i could see someone coming by later and just combining the two lines because they think this is a mistake :P 

for the record i don't see a problem with it, just curious why it's needed

Reply:
the "alternative" is the linter forcing each field in the destructure into its own line which I think looks worse

Reply:
actually this isn't an issue anymore since I moved the spam check outside of this component